applications/zigbee_weather_station/src/sensor.h

```diff
@@ -9,6 +9,17 @@
 
 #include <drivers/sensor.h>
 
+/* Measurements ranges for Bosch BME688 sensor */
+#define SENSOR_TEMP_CELSIUS_MIN (-40)
+#define SENSOR_TEMP_CELSIUS_MAX (85)
+#define SENSOR_TEMP_CELSIUS_TOLERANCE (1)
+#define SENSOR_PRESSURE_KPA_MIN (30)
+#define SENSOR_PRESSURE_KPA_MAX (110)
+#define SENSOR_PRESSURE_KPA_TOLERANCE (0)
+#define SENSOR_HUMIDITY_PERCENT_MIN (10)
+#define SENSOR_HUMIDITY_PERCENT_MAX (90)
+#define SENSOR_HUMIDITY_PERCENT_TOLERANCE (3)
```

Comment:
If humidity's tolerance is not supported, I would suggest to remove it as well.

Reply:
Humidity tolerance is not supported on ZCL cluster level, the sensor supports it hence I left it intentionally.
It will require less investigation in the sensor's datasheets after cluster is modified, which is planned.